mission.sqm

```diff
@@ -572,6 +589,7 @@ class Mission
 					{
 						skill=0.60000002;
 						rank="SERGEANT";
+						init="[west, alpha1] call BIS_fnc_addRespawnPosition;";
```

Comment:
Remove these lines from the objects, add itt to initPlayerServer.sqf and/or/xor onPlayerRespawn.sqf. Former should do the trick. Refer to https://community.bistudio.com/wiki/Event_Scripts and https://community.bistudio.com/wiki/BIS_fnc_addRespawnPosition and https://community.bistudio.com/wiki/Initialization_Order